docs/background/glossary.md

```diff
@@ -163,6 +169,8 @@ Please note that this Glossary is **work in progress**. Do you encounter missing
 * SV: see Structural Variant.
 
 * trans: two variants are **"in trans"** when they are on different alleles (DNA molecules, chromosomes).
+
+* transcript segment: a representation of a segment of transcribed sequence denoted by a 5’ and 3’ boundary.
```

Comment:
per @jtdendunnen:
> Do we need this definition? Is the definition "a representation of a segment of transcribed sequence denoted by a 5’ and 3’ boundary" in fact the definition of "exon" and "intron"?

Reply:
A transcript segment is an RNA molecule. I can see where the confusion stems from and will drop this definition in alignment with other requested changes.

Reply:
addressed in faa35ca